test/plugin/base_test.rb

```diff
@@ -307,6 +307,19 @@ def setup
     }
   }
 
+  HTTP_REQUEST_MESSAGE = {
+    'requestMethod' => 'POST',
+    'requestUrl' => 'http://example/',
+    'requestSize' => 210,
+    'status' => 200,
+    'responseSize' => 65,
+    'userAgent' => 'USER AGENT 1.0',
+    'remoteIp' => '55.55.55.55',
+    'referer' => 'http://referer/',
+    'cacheHit' => false,
+    'validatedWithOriginServer' => true
```

Comment:
The current official name is `'cacheValidatedWithOriginServer'`, isn't it? Let's keep that name in the base class, and rename in the v1 tests, instead of having to accommodate this in the gRPC test.

To be clear, here's what I'm suggesting:
```
module BaseTest
  HTTP_REQUEST_MESSAGE = {
    'requestMethod' => 'POST',
    'requestUrl' => 'http://example/',
    'requestSize' => 210,
    'status' => 200,
    'responseSize' => 65,
    'userAgent' => 'USER AGENT 1.0',
    'remoteIp' => '55.55.55.55',
    'referer' => 'http://referer/',
    'cacheHit' => false,
    'cacheValidatedWithOriginServer' => true
  }
  def http_request_message
    HTTP_REQUEST_MESSAGE
  end
end
class GoogleCloudOutputTest < Test::Unit::TestCase
  include BaseTest
  def http_request_message
    super.merge('validatedWithOriginServer' => super['cacheValidatedWithOriginServer'])
      .reject do |k, _| k == 'cacheValidatedWithOriginServer' end
  end
end
```
(You could even define a `rename_key` helper function to do the above:
```
def rename_key(hash, old_key, new_key)
  hash.merge(new_key => hash[old_key]).reject do |k, _| k == old_key end
end
```
and then the above override of `http_request_message` becomes:
```
def http_request_message
  rename_key(super, 'validatedWithOriginServer', 'cacheValidatedWithOriginServer')
end
```
).